### PR TITLE
feat: add DA for fscloud

### PR DESCRIPTION
**Pull request by akocbek**
### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by ocofaigh** [comment]
There is a breaking change coming in https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/pull/178 so going to merge that first

**Comment by ocofaigh** [comment]
@akocbek Daniel PR is now merged, so you should be able to proceed now. Thanks

File: solutions/fscloud/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# IBM Event Streams that meets Financial Services Cloud requirements
+
+This architecture creates Event Streams that meets Financial Services Cloud requirements and supports provisioning the following resources:
+- A resource group, if one is not passed in.
+- IBM Event Streams supporting Financial Services Cloud requirements
+
+![da-fscloud](../../reference-architecture/da-fscloud.svg)
```
[the remark this answers is not in the document]
Comment (akocbek) [reply solutions/fscloud/README.md:11]: cloudant has the same and it works https://github.com/terraform-ibm-modules/terraform-ibm-cloudant/blob/main/solutions/dedicated/README.md?plain=1#L9

File: solutions/fscloud/version.tf [hunk 2]
```diff
@@ -1,10 +1,9 @@
 terraform {
   required_version = ">= 1.3.0, <1.6.0"
   required_providers {
-    # Use latest version of provider in non-basic examples to verify latest version works with module
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = ">= 1.56.1"
+      version = "1.56.1"
```
Comment (ocofaigh) [inline solutions/fscloud/version.tf:6]: Can you update this to 1.62.0 since its the latest and renovate will always use bump this to latest since its a DA

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@akocbek just 1 comment and we should be good

**Review by ocofaigh — APPROVED** [review]
Merging without a release for now, and will release when ibm_catalog.json is added

**Pull request merged**